src/Sylius/Bundle/ShopBundle/Resources/private/js/sylius-variants-prices.js

```diff
@@ -9,17 +9,15 @@
 
 import $ from 'jquery';
 
-function formatAppliedPromotions(appliedPromotions, locale = 'en_US') {
+function formatAppliedPromotions(appliedPromotions) {
   let appliedPromotionsElement = '';
   $('#appliedPromotions').html('');
 
   if (appliedPromotions !== '[]') {
     $.each(appliedPromotions, (index, promotion) => {
-      if (promotion['translations'][locale].description !== null) {
-        appliedPromotionsElement += `<div class="ui blue label promotion_label" style="margin: 1rem 0;"><div class="row ui small sylius_catalog_promotion">${promotion['translations'][locale].name} - ${promotion['translations'][locale].description}</div></div>`;
-      } else {
-        appliedPromotionsElement += `<div class="ui blue label promotion_label" style="margin: 1rem 0;"><div class="row ui small sylius_catalog_promotion">${promotion['translations'][locale].name}</div></div>`;
-      }
+      let promotionInfo = promotion.name;
```

Comment:
here we should display, translatable field - `label` instead of name  
```suggestion
      let promotionInfo = promotion.label;
```